examples/bzlmod/MODULE.bazel

```diff
@@ -11,17 +11,21 @@ local_path_override(
     path = "../..",
 )
 
+PYTHON_NAME = "python3_9"
+
+PYTHON_TOOLCHAINS = PYTHON_NAME + "_toolchains"
+
 python = use_extension("@rules_python//python/extensions:python.bzl", "python")
 python.toolchain(
-    name = "python3_9",
+    name = PYTHON_NAME,
     configure_coverage_tool = True,
     python_version = "3.9",
 )
-use_repo(python, "python3_9")
-use_repo(python, "python3_9_toolchains")
+use_repo(python, PYTHON_NAME)
+use_repo(python, PYTHON_TOOLCHAINS)
 
 register_toolchains(
-    "@python3_9_toolchains//:all",
+    "@{}//:all".format(PYTHON_TOOLCHAINS),
```

Comment:
nit: I find the usage of constants here making it harder to read. Even though the definition and the usage of the constants is nearby, my eyes need to jump to lines 14 and 16 when reading line 28. Having the text verbatim would be better at the expense of very little duplication.

Reply:
If you look at the example now, we now have multiple versions, and I find that having variables causes me less grief because of typos.  Do you have a compromise?